guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,159 @@
+enabled=1
+gpgcheck=1
+
+[leapp-katello]
+name=Katello 4.4
+baseurl=https://yum.theforeman.org/katello/4.4/katello/el8/$basearch/
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+
+[leapp-katello-candlepin]
+name=Candlepin: an open source entitlement management system.
+baseurl=https://yum.theforeman.org/katello/4.4/candlepin/el8/$basearch/
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+
+[leapp-pulpcore]
+name=pulpcore: Fetch, Upload, Organize, and Distribute Software Packages.
+baseurl=https://yum.theforeman.org/pulpcore/3.16/el8/$basearch/
+gpgkey=https://yum.theforeman.org/pulpcore/3.16/GPG-RPM-KEY-pulpcore
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+----
+
+* If you are not using {Project}, you can leave out `leapp-katello`, `leapp-katello-candlepin` and `leapp-pulpcore`.
+
+* If you are using Puppet 6 instead of Puppet 7, replace the `7` with a `6` in the `leapp-puppet7` entry.
+
+* You need a Puppet repository for the Puppet agent that the installer is using.
+
+. We do not support EL8 installations with EPEL8 enabled, so remove `epel-release`:
++
+----
+# yum remove epel-release
+----
+. Let LEAPP analyze your system:
++
+----
+# leapp preupgrade
+----
+
++
+
+The first run is expected to fail but report issues.
+Continue to the next step for remediation.
+
+. Fix issues found by LEAPP (check `/var/log/leapp/leapp-report.txt` for details):
++
+----
+# rmmod pata_acpi
+# echo PermitRootLogin yes | tee -a /etc/ssh/sshd_config
+# leapp answer --section remove_pam_pkcs11_module_check.confirm=True
+----
++
+
+** If `preupgrade` aborts with a dependency resolution error such as:
++
+----
+# package rubygem-fx-0.5.0-2.el8.noarch requires rubygem(railties) >= 4.0.0, but none of the providers can install
+----
+or:
++
+----
+ # package rubygem-railties-6.0.4.7-1.el8.noarch requires rubygem(thor) < 2.0, but none of the providers can install
+----
+
+** Remove packages that are no longer available from any repository:
++
+----
+# dnf remove $(dnf repoquery --extras --exclude '*katello*' --exclude '*pulp*' --exclude '*localhost*' --exclude "*$HOSTNAME*" --exclude libmodulemd) –
+----
+
++
+
+** Check the output if it does not contain any local packages.
+
+** Remove `rubygem-thor`:
++
+----
+# yum remove rubygem-thor
+----
+
+. Ensure `leapp preupgrade` has no issues.
+. Run:
++
+----
+# leapp upgrade
+----
+
++
+
+. Reboot the system.
+. After the system reboots, a live system conducts the upgrade.
+. Reboot to fix SELinux labels.
+. Reboot into the final {RHEL} 8 system.
+. After the system reboots, LEAPP finishes the upgrade.
+. You can watch LEAPP finish the upgrade with:
++
+----
+# journalctl -u leapp_resume -f
+----
+
+[WARNING]
+====
+If you installed the system and had to use `--disable-system-checks`, the last step of the upgrade will fail, and you will need to call `foreman-installer --disable-system-checks` manually once the system is booted up.
```

Comment:
this should use the `{foreman-installer}` macro

Reply:
Also, we don't use language like `booted up` in official docs. 
IBM Style Guide has specifications for when and if you ever use the word "once". 
We do  not use the future tense in our docs. I've mentioned this several times by now. 
We do not use warnings in our docs unless the action with irreparably damage the deployment or cause harm to human life. This does not seem to be the case here.

Reply:
Understood @melcorr T

There are other instances in the existing documentation where **NOTE** and **WARNING** have future tense in the text so I wasn't sure if that was the only time it was allowed.